sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_common/_response_handlers.py

```diff
@@ -23,34 +23,25 @@
 # IN THE SOFTWARE.
 #
 # --------------------------------------------------------------------------
-from typing import Any, Dict
+import json
 
-from azure.core.pipeline import PipelineResponse
-
-from .._generated.models import SchemaId as InternalSchemaId
-from ._schema import SchemaProperties, Schema
+from azure.core.rest import HttpResponse
 
 
-def _parse_response_schema_id(pipeline_response, deserialized, response_headers):  # pylint: disable=unused-argument
-    # type: (PipelineResponse, InternalSchemaId, Dict[str, Any]) -> SchemaProperties
-    """
-
-    :param pipeline_response:
-    :param deserialized:
-    :param response_headers:
-    :return:
-    """
-    return SchemaProperties(schema_id=deserialized.id, **response_headers)
+from ._schema import SchemaProperties, Schema
 
 
-def _parse_response_schema(pipeline_response, deserialized, response_headers):  # pylint: disable=unused-argument
-    # type: (PipelineResponse, str, Dict[str, Any]) -> Schema
-    """
+def _parse_response_schema_id(response):
+    # type: (HttpResponse) -> SchemaProperties
+    return SchemaProperties(
+        schema_id=response.json()["id"],
+        **response.headers
+    )
 
-    :param pipeline_response:
-    :param deserialized:
-    :param response_headers:
-    :return:
-    """
 
-    return Schema(schema_content=deserialized, schema_properties=SchemaProperties(**response_headers))
+def _parse_response_schema(response):
+    # type: (HttpResponse) -> Schema
+    return Schema(
+        schema_content=json.dumps(response.json()),
```

Comment:
qq: need to call `json.dumps(response.json())` to get rid of the escape backslash in the response content, is this the right way to do it?